flash/core/data/transforms.py

```diff
@@ -13,11 +13,61 @@
 # limitations under the License.
 from typing import Any, Dict, Mapping, Sequence, Union
 
+import numpy as np
 import torch
-from torch import nn
+from torch import nn, Tensor
 
+from flash.core.data.io.input import DataKeys
 from flash.core.data.utilities.collate import default_collate
 from flash.core.data.utils import convert_to_modules
+from flash.core.utilities.imports import _ALBUMENTATIONS_AVAILABLE, requires
+
+if _ALBUMENTATIONS_AVAILABLE:
+    from albumentations import BasicTransform, Compose
+    from albumentations.pytorch import ToTensorV2
+else:
+    BasicTransform, Compose, ToTensorV2 = object, object, object
+
+
+class AlbumentationsAdapter(nn.Module):
+    # mapping from albumentations to Flash
+    TRANSFORM_INPUT_MAPPING = {"image": DataKeys.INPUT, "mask": DataKeys.TARGET}
+
+    @requires("albumentations")
+    def __init__(
+        self,
+        transform: Union[BasicTransform, Sequence[BasicTransform]],
+        mapping: dict = None,
+        image_key: str = DataKeys.INPUT,
+    ):
+        super().__init__()
+        if not isinstance(transform, (list, tuple)):
+            transform = [transform]
+        self.transform = Compose(list(transform) + [ToTensorV2()])
```

Comment:
The added `ToTensorV2` here seems strange to me. Would prefer for the user to manually pass the tensor transform if that is something they want.

Reply:
well, I found that you pass it in each sequence so was thinking to abstract it... but if you prefer to have added by the user always, I do not mind... :rabbit: